<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LambdaSnail::Juno::budget::LSBudgetCategoryBar</class>
 <widget class="QWidget" name="LambdaSnail::Juno::budget::LSBudgetCategoryBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSBudgetCategoryBar</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="categoryLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="categoryLimitVisual">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="categoryLimit">
     <property name="maximum">
      <double>99999999999999.984375000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
